### ISSUE TITLE
About 1-shot and 5-shot (from more detailed implementation)

### ISSUE THREAD

**Issue by ZhangYikaii** [leokarlin/LaSO#14]
Hi.
Thank you for sharing the code, It's wonderful.

The difference between 1-shot and 5-shot setting's **sampling** is only [here](https://github.com/leokarlin/LaSO/blob/master/oneshot/alfassy/datasets.py#L695).
Why in the case of 5-shot, **divide by 5** in each batch of `fakeVectorsPairs`?

(It doesn't matter that your implementation is different from the Episodic Sampling in Few-Shot classical methods like Prototypical Networks, but `5-shot` means five samples per class. So how to explain division 5 [here](https://github.com/leokarlin/LaSO/blob/master/oneshot/alfassy/datasets.py#L695)?)

Thank you again and I am looking forward to your reply.

**Comment by alfassy** [comment]
Hi,
We don't work in episodes here, we train with our 5 shot labeled samples, then we just check performance on the entire validation set, if I recall correctly I changes the batch size in the 5 shot case because of CUDA memory limitations, I was using K40 gpu at that time.
Regards,
Amit

**Comment by ZhangYikaii** [comment]
Thank you very much!
